packages/sil-voting-app/src/routes/dashboard/components/Proposal/StatusPending.tsx

```diff
@@ -0,0 +1,38 @@
+import Block from 'medulas-react-components/lib/components/Block';
+import Img from 'medulas-react-components/lib/components/Image';
+import Typography from 'medulas-react-components/lib/components/Typography';
+import React from 'react';
+import { ReadonlyDate } from 'readonly-date';
+
+import deleteIcon from '../../../../assets/delete.svg';
+
+interface Props {
+  readonly expiryDate: Date;
+}
+
+const StatusPending = (props: Props): JSX.Element => {
+  return (
+    <React.Fragment>
+      <Block display="flex" alignItems="center" marginBottom={1}>
+        <Typography variant="body1" weight="semibold">
+          Expires on {`${(props.expiryDate as ReadonlyDate).toLocaleString()}`}
+        </Typography>
+        <Block marginLeft={2}>
+          <Block display="flex" alignItems="center">
+            <Img src={deleteIcon} alt="Delete Icon" height="16px" />
+            <Typography variant="body1" weight="semibold">
+              Delete
+            </Typography>
+          </Block>
+        </Block>
+      </Block>
+      <Block padding={1} bgcolor="#d8d8d8" borderRadius="16px">
```

Comment:
Do not use constants directly here, you have two options:
1. Use useTheme from material-ui and access a color prop from there.
2. Create theme/css.ts file where you place the constants with a readable name like BOX_BACKGROUND

Generally speaking is better to use 1, because if you change the theme, automatically you have changed the color, but if you are using a very specific one which is not related with the theme, feel free to use second option.

Reply:
I used approach 2 because the two colors that I used are not defined in the theme.